canvas_modules/harness/test_resources/parameterDefs/toggle_paramDef.json

```diff
@@ -0,0 +1,433 @@
+{
+  "titleDefinition": {
+    "title": "toggle Fields"
+  },
+  "current_parameters": {
+    "toggleـstring_null": true,
```

Comment:
Can you cleanup the naming here?  These are also wrong.  For example this one should be "toggle_null": null.  For undefined there wouldn't be a current parameter.

Also remove `string` from all the names since it doesn't make sense.

Reply:
Updated `toggle_null` & removed string